templates/kpi/deployment.yaml

```diff
@@ -36,17 +36,17 @@ spec:
             - name: http
               containerPort: 8000
               protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /accounts/login/
-              port: http
-            initialDelaySeconds: 5
-            timeoutSeconds: 5
-          readinessProbe:
-            httpGet:
-              path: /accounts/login/
-              port: http
-            initialDelaySeconds: 5
+{{/*          livenessProbe:*/}}
```

Comment:
Was this a mistake?

Reply:
Yes, sorry about that. I'll fix this

Reply:
> Was this a mistake?

This is fixed along with the other erroneous changes